src/presentation/http/router/join.test.ts

```diff
@@ -3,16 +3,61 @@ import { describe, test, expect } from 'vitest';
 describe('Join API', () => {
   describe('POST /join/:hash', () => {
     test('Returns 406 when user is already in the team', async () => {
-      const hash = 'Hzh2hy4igf';
-      const userId = 2;
+      const invitationHash = 'Hzh2hy4igf';
+      const userId = 1;
       const accessToken = global.auth(userId);
 
-      const response = await global.api?.fakeRequest({
+      /**
+       * truncate all tables, which are needed
+       * restart autoincrement sequences for data to start with id 1
+       *
+       * TODO get rid of restarting database data in tests (use restart-database-data script in beforeEach)
+       */
+      await global.db.truncateTables();
+
+      /** create test user */
+      await global.db.insertUser({
+        email: 'testemal@CodeXmail.com',
+        name: 'CodeX',
+      });
+
+      /** create test note for created user */
+      await global.db.insertNote({
+        creatorId: 1,
```

Comment:
`1` looks like a magic number. It's better to use `user.id` where `user` returned by `insertUser`.